pkg/clients/openshift.go

```diff
@@ -0,0 +1,198 @@
+package clients
```

Comment:
can we use https://github.com/openshift/client-go or https://github.com/openshift/kubernetes-client-go or some other openshift/k8s client go api library?